controllers/status_controller.go

```diff
@@ -113,6 +113,8 @@ func (r *StatusReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		log.Info("unable to fetch the corresponding Service of type LoadBalancer")
 	}
 
+	// TODO also update the port/ip of databases mentioned in owner.Spec.PostgresConnectionInfo so that e.g. CWNP are always up to date
```

Comment:
review and delete if not required anymore

Reply:
I think we still need this.

When the Status of the ressource changes, we also update the ip and port of the database.

If there is a connected database, the configured ip and port are not updated currently. So in the rare case a postgres gets another port (or even another ip), the connected postgres has an outdated configuration. this would only be updated if someone made changes to that connected postgres' custom ressource.